docs/realtime.md

```diff
@@ -0,0 +1,145 @@
+# Realtime docs
+
+## Context
+
+### Definitions
+
+- **Events feed:** the feed of occurring events. Most of them will come from couchdb, some jobs might also trigger them. There is two types of feeds:
+  - **continuous** allows to follow events as they occurs
+  - **interval** allow the see the history of the feed from any given time
+
+
+- **Realtime:** user experienced updates of the interface from change happening from another source. **Ie,** *I have a folder opened in cozy-files on the browser, I take some pictures from my smartphone, the pictures appears in the folder without me needing to refresh the browser tab.*
+
+### What couchdb offers
```

Comment:
We also have events that are not coming from CouchDB:

- when the user closes its session
- the jobs system